src/main/java/it/pagopa/transactions/services/v1/TransactionsService.java

```diff
@@ -273,7 +273,8 @@ private TransactionOutcomeInfoDto buildTransactionOutcomeInfoDtoFromView(BaseTra
                     TransactionOutcomeInfoDto.OutcomeEnum outcome = evaluateOutcome(transaction.getStatus(), transaction.getSendPaymentResultOutcome(), transaction.getPaymentGateway(), transaction.getGatewayAuthorizationStatus(), transaction.getAuthorizationErrorCode());
                                 return new TransactionOutcomeInfoDto()
                                 .outcome(outcome)
-                                .totalAmount(outcome == TransactionOutcomeInfoDto.OutcomeEnum.NUMBER_0 ? transaction.getPaymentNotices().stream().mapToInt(it.pagopa.ecommerce.commons.documents.PaymentNotice::getAmount).sum() + Optional.ofNullable(transaction.getFeeTotal()).orElse(0) : null)
+                                .totalAmount(outcome == TransactionOutcomeInfoDto.OutcomeEnum.NUMBER_0  ? transaction.getPaymentNotices().stream().mapToInt(it.pagopa.ecommerce.commons.documents.PaymentNotice::getAmount).sum() : null)
```

Comment:
🚲 
we could calculate total amount as the amount that take in charge also the fees and then return total amount and fees (as we do in close payment request [here](https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/blob/e4079325a63768277aeacd10d44ab9e23d218389/src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/services/v2/NodeService.kt#L257C14-L259))
what do you think?

Reply:
We decided to hold this structure and update the description:

- totalAmount is about the total amount paid fees excluded
- fees is about the amount paid only for fees
- both are in eurocents